### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by aapeliv**

### CODE REVIEW

File: app/frontend/src/features/constants.ts [hunk 1]
```diff
@@ -158,15 +158,17 @@ Use the **map** to find a host, or **search by name** in the search bar (if you
...
-* Reworked and simplified the signup process
-* Moved the contributor form to the app
-* Simplified the bug reporting process
+* Redesigned the search feature to display results on the map (filters coming soon)
+* Added the ability to delete, ban, and block users (email us if you'd like to be deleted or to ban someone)
+* Implemented onboarding emails to welcome new users
+* Updated the Terms of Service
+* Implemented nicer error messages for frontend bugs
+* Updated the contributor form
+* Fixed internal bugs including with coordinate wrapping and speeding up CI/CD by eliminating duplicate tests due to backend refactoring
```
Comment (lucaslcode) [inline app/frontend/src/features/constants.ts:171]: ......

**Review by lucaslcode — APPROVED** [review]

File: app/frontend/src/features/constants.ts (hunk 1 above)
Comment (aapeliv) [reply app/frontend/src/features/constants.ts:171]: huh?

Comment (lucaslcode) [reply app/frontend/src/features/constants.ts:171]: This one is a bit technical for end users, no? It's not a big deal though

Comment (aapeliv) [reply app/frontend/src/features/constants.ts:171]: Good to put one technical thing in there so people know we're doing very fancy hardcore things under the hood ;)

**Pull request merged**